docgen/src/examples/tourism/App.js

```diff
@@ -234,11 +239,15 @@ const RoomType = RefinementList.connect(({items, refine, selectedItems}) => {
 });
 
 function Price() {
+  const theme = _.entries(Range.defaultTheme).reduce((acc, i) => {
```

Comment:
Yes, we could provide two utility method 👍 
* an erase merge 
* a combined merge 

In the meantime, updating the docs is fine.